src/isaac-data-types.d.ts

```diff
@@ -123,7 +123,7 @@ export interface IsaacEventPage extends Content {
   location?: Location;
   preResources?: ExternalReference[];
   postResources?: ExternalReference[];
-  eventSurvey?: ExternalReference[];
+  eventSurvey?: string;
```

Comment:
Looks like you have manually modified the types file here? You'll need to make sure the changes are in the API and then regenerate the file using the mvn extension